### DISCUSSION TITLE
How to do checkouts in githup desktop

### DISCUSSION THREAD

**Discussion by jaumesagues** [community/community#39255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I would like to know how I can come back to and old commit with GitHub Desktop.

I know how can I do with Git:

git checkout <id commit>

However, I do not know if it is possible to do with GitHub Desktop and how I can do

**Comment by steveward** [comment]
@jaumesagues thanks for the question! GitHub Desktop allows you to create a branch off of a previous commit in your repository's history. You can do this by right-clicking a commit in the `History` view and selecting `Create Branch from Commit`. The ability to checkout a previous commit without needing to create a branch has also been requested, and is being tracked in https://github.com/desktop/desktop/issues/10068. You can follow along there for any updates about this. Let me know if you have any further questions!
